### ISSUE TITLE
Write some unit & integration tests for `Contact` page to avoid regressions

### ISSUE THREAD

**Issue by zainfathoni** [kawalcovid19/wargabantuwarga.com#414]

**Comment by fauzialz** [comment]
Hello, I have a little experience in writing e2e testing by using Cypress. Is there any consideration on using some particular library to do the e2e?

**Comment by zainfathoni** [comment]
> Hello, I have a little experience in writing e2e testing by using Cypress. Is there any consideration on using some particular library to do the e2e?

The only considerations are:

1. It should work well with Cypress
2. It should comply with this testing [guiding principles](https://testing-library.com/docs/guiding-principles) defined by the Testing Library community

**Comment by iamyuu** [comment]
Hi, @fauzialz do you wanna take this ticket?

**Comment by fauzialz** [comment]
> Hi, @fauzialz do you wanna take this ticket?

Hello. Sorry for late response. You can take it if you want 👍

**Comment by shyambh** [comment]
Hi @zainfathoni , I can write a few e2e tests for the Contact page. Is /kontak-darurat the contact page? And what sort of tests are you expecting?
Thanks.

**Comment by zainfathoni** [comment]
> Hi @zainfathoni , I can write a few e2e tests for the Contact page. Is /kontak-darurat the contact page? And what sort of tests are you expecting?
> Thanks.

No, `Contact` page is the details page for each entry in a given province.
e.g., https://www.wargabantuwarga.com/provinces/dki-jakarta/bantuan-kebutuhan-pokok-layanan-antar-pangan-gratis-banten-act-humanity-care-line-08001165228

You can refer to the [unit testing scenarios for the page](https://www.wargabantuwarga.com/provinces/dki-jakarta/bantuan-kebutuhan-pokok-layanan-antar-pangan-gratis-banten-act-humanity-care-line-08001165228) and come up with your own end-to-end testing scenarios.

**Comment by zainfathoni** [comment]
Btw, this project is considered dormant.
If you want to contribute to a recent project that I am actively maintaining at the moment, you can look at this project instead: https://github.com/zainfathoni/senar.ai.

**Comment by shyambh** [comment]
Ok. I will look into that project instead.
